scripts/canisters/local_deploy/start_upgrades.sh

```diff
@@ -0,0 +1,29 @@
+# !/bin/bash
+
+
+dfx build individual_user_template
+
+# Specify the path to your Wasm.gz file
+wasm=".dfx/ic/canisters/individual_user_template/individual_user_template.wasm.gz"
```

Comment:
Don't we have to gzip here like we were doing previously or does dfx automatically do that. I see the wasm path is different here than the other ones. Any reason ?